__tests__/app.test.js

```diff
@@ -151,3 +154,57 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("POST:201 inserts a new comment to the comments table and sends the new comment back to the client", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "I like this article",
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment.comment_id).toBe(19);
+        expect(body.comment.article_id).toBe(2);
+      });
+  });
+  test("POST:400 responds with an appropriate status and error message when provided with a bad comment (no user name)", () => {
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send({
+        body: "I like this article",
+      })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
```

Comment:
Nice test, but there should be another test for when a username is given but one that does not exist in the database i.e. "banana" 

What should be given as a response in that case?